feature_engine/feature_selection.py

```diff
@@ -88,3 +89,138 @@ def transform(self, X):
         return X
 
 
+class SelectFeatures(TransformerMixin, BaseEstimator):
+    """
+    SelectFeatures()
+
+    Drops constant and quasi constant variables from a dataframe.
+    Constant variables are those variables which show only one value for all the observations.
+    By default, all constant variables are dropped.
+    Quasi-constant variables are those variables which are almost constant. In other words, those features have the same
+    value for large subset of total observations.
+    This transformer works for both, numerical and categorical variables.
+
+    Parameters
+    ----------
+
+    drop_quasi_constant_features : bool, default=False
+        Whether to drop quasi constant features
+
+    quasi_constant_threshold: float, default=None
+        When drop_quasi_constant_features is True, features showing quasi_constant_threshold percentage of values
+        will be dropped
+
+    Attributes
+    ----------
+
+    constant_features_: list
+        List of constant features in a dataframe
+
+    quasi_constant_features_: list
+        List of quasi constant features in a dataframe
+
+    selected_features_: list
+        List of selected features which are non-constant and non-quasi constant
+
+    """
+    def __init__(self, drop_quasi_constant_features=False, quasi_constant_threshold=None):
+
```

Comment:
init parameters should be threshold, with value 1 by default to remove only constant variables. Threshold can also take values between 0 and 1 to remove quasi-constant.

Another parameters for init is variables, which gives the user the opportunity to select which variables they want examined and removed. By default, the parameter takes None, and would select all variables in the dataframe.